pkg/util/common.go

```diff
@@ -208,8 +208,6 @@ func CreateHTTPTransport() *http.Transport {
 			Timeout: 30 * time.Second,
 			// Enables TCP keepalives to detect broken connections
 			KeepAlive: 30 * time.Second,
-			// Enables happy eyeballs
-			DualStack: true,
```

Comment:
can we set it to `false` explicitly, and add a comment on why? It may become deprecated in go `1.12`, but IMO actually setting it here will still help us when we upgrade to go 1.12 in the future